php_docs_deploy.sh

```diff
@@ -1,14 +1,23 @@
 #!/bin/sh
 
 # version branch names in descending order
-VERSIONS=(master v1.9 v1.8 v1.7 v1.6 v1.5 v1.4 v1.3 v1.2 v1.1)
+VERSIONS=(master v1.15 v1.14 v1.13 v1.12 v1.11 v1.10 v1.9 v1.8 v1.7 v1.6 v1.5 v1.4 v1.3 v1.2 v1.1)
 
 # clean
 git checkout master
 make clean
```

Comment:
Issue:
"clean" is no longer a target, so it produces an error.

Suggestion:
Remove this part of the script.